### PR TITLE
Add Sku parameter for Set-AzureResource and New-AzureResource

### PR DESCRIPTION
**Pull request by chidmdxx**
This add support for the Sku property for NewAzureResource and SetAzureResource and updates the tests to support it

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@chidmdxx__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Entities/Resources/ResourceSku.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.ResourceManager.Cmdlets.Entities.Resources
+{
+    using Newtonsoft.Json;
```
Comment (stankovski) [inline src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Entities/Resources/ResourceSku.cs]: using should be above namespace

File: src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/ResourceTests.ps1 [hunk 2]
```diff
@@ -27,9 +27,9 @@ function Test-CreatesNewSimpleResource
 	$resourceType = "Microsoft.Sql/servers"
 
 	# Test
-	New-AzureRmResourceGroup -Name $rgname -Location $rglocation
-	$actual = New-AzureRmResource -Name $rname -Location $location -Tags @{Name = "testtag"; Value = "testval"} -ResourceGroupName $rgname -ResourceType $resourceType -PropertyObject @{"administratorLogin" = "adminuser"; "administratorLoginPassword" = "P@ssword1"} -ApiVersion $apiversion
-	$expected = Get-AzureRmResource -Name $rname -ResourceGroupName $rgname -ResourceType $resourceType -ApiVersion $apiversion
+	New-AzureRMResourceGroup -Name $rgname -Location $rglocation
```
Comment (stankovski) [inline src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/ResourceTests.ps1]: Nit: Should be AzureRm not AzureRM

**Comment by stankovski** [comment]
Please address these minor issues in the future PRs. Merging.

**Pull request merged**